src/main/java/com/google/sps/servlets/PostQuestionServlet.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/** 
+   * This servlet will post a question to the forum.
+   */
+@WebServlet("/post-question")
+public class PostQuestionServlet extends HttpServlet {
+  // All the variables needed to connect to the local database.
+  // P.S.: Change the timezone if needed (https://github.com/dbeaver/dbeaver/wiki/JDBC-Time-Zones).
+  String url = "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=America/Mexico_City";
+  String user = "root";
+  String password = "";
```

Comment:
Forgot to tell you guys, but Gloria suggested we should use constants from the Utility class, so I added these:
```
static final String SQL_URL = "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=PST8PDT";
static final String SQL_USER = "root";
static final String SQL_PASSWORD = "";
```
You could update to these.

Reply:
i'll wait for the fix that makes them public, but yeah that will make things easier!

Reply:
Sounds good!

Reply:
To avoid ending up with a huge Utility class, feel free to make a SqlConstants class, and other Constants classes as needed.